python/TestHarness/util.py

```diff
@@ -552,6 +552,14 @@ def getIfAsioExists(moose_dir):
         option_set.add('FALSE')
     return option_set
 
+def getIfLibtorchExists(moose_dir):
+    option_set = set(['ALL'])
+    if os.path.exists(f'{moose_dir}/modules/stochastic_tools/libtorch'):
```

Comment:
We should debate the location of this...

Reply:
Okay, I've also been contemplating on actually putting it into the framework instead of a module. People from other modules might want to use it in the future. I'll bring this up today in the DM.